django_project/social_users/templates/social_users/profile.html

```diff
@@ -97,10 +97,19 @@ <h4 class="modal-title">Your API KEY</h4>
                             <div class="modal-body">
                                 {% if api_keys %}
                                     {% for api_key in api_keys %}
-                                        <div>{{ api_key }}</div>
+                                        <div>
+                                            {{ api_key.api_key }}
+                                            {% if not api_key.is_active %}
+                                                <span>(Waiting admin approval)</span>
+                                            {% endif %}
+                                        </div>
                                     {% endfor %}
                                 {% else %}
-                                    <i style="color: #f44a52">You don't have api key. Please update profile or ask administrator.</i>
+                                    <a href="{% url 'enrollment-form' %}">
+                                        <button type="button" class="btn btn-primary">
+                                            You don't have any api key, do enrollment by clicking here.
```

Comment:
You don't have any API keys issued to you. You can enroll for a new API key by clicking here: